partner_industry_secondary/models/res_partner_industry.py

```diff
@@ -28,11 +29,25 @@ def get_names(cat):
             """Return the list [cat.name, cat.parent_id.name, ...]"""
             res = []
             while cat:
-                res.insert(0, cat.name)
+                res.append(cat.name)
                 cat = cat.parent_id
             return res
 
-        return [(cat.id, " / ".join(get_names(cat))) for cat in self]
+        if (
+            self.env["ir.config_parameter"]
+            .sudo()
+            .get_param("partner_industry_secondary.display_last_child_first")
+        ):
+            # Display last child first
+            result = []
+            for cat in self:
+                cat_name, *parent_cats = get_names(cat)
+                if parent_cats:
+                    cat_name = f"{cat_name} ({' < '.join(parent_cats)})"
+                result.append((cat.id, cat_name))
+            return result
+        # Default display (Ancestor / Grandpa / Dad / Child)
```

Comment:
suggestion: just for the sake of uniformity

```suggestion
        # Default display (Grandparent / Parent / Child)
```